core/templates/pages/admin-page/misc-tab/admin-misc-tab.component.html

```diff
@@ -14,6 +14,11 @@ <h3>Search Index</h3>
   <button (click)="clearSearchIndex()">Clear Search Index</button>
 </mat-card>
 
+<mat-card class="oppia-admin-page-card oppia-page-card oppia-long-text">
+  <h3>Publish the chapters of the Length Measurements topic</h3>
+  <button (click)="publishChapters()">Publish Chapters</button>
```

Comment:
Update name of function to include length measurements in the function name.

Function names should fully describe/scope what they're doing.

Reply:
Done!